dockerfiles/extract_diffs/Dockerfile

```diff
@@ -1,48 +0,0 @@
-FROM golang:alpine as builder
```

Comment:
Probably good to remove the [build/deploy](https://github.com/makerdao/vulcanizedb/blob/staging/.travis/deploy.sh#L32) here as well

Reply:
good catch!